test/hf/run_glue_hpo.py

```diff
@@ -0,0 +1,42 @@
+import sys
+sys.path.insert(0, "../../")
+from flaml.nlp.autohf import AutoHuggingFace
```

Comment:
in flaml/nlp/__init__.py, import AutoHuggingFace.
here: 
`from flaml.nlp import AutoHuggingFace`